responses.py

```diff
@@ -527,16 +527,20 @@ def add(
 
     def add_passthru(self, prefix):
         """
-        Register a URL prefix to passthru any non-matching mock requests to.
+        Register a URL regex to passthru any non-matching mock requests to.
 
         For example, to allow any request to 'https://example.com', but require
         mocks for the remainder, you would add the prefix as so:
 
         >>> responses.add_passthru('https://example.com')
+
+        Regex can be used like:
+
+        >>> responses.add_passthru('https://example.com/\\w+')
         """
         if _has_unicode(prefix):
             prefix = _clean_unicode(prefix)
-        self.passthru_prefixes += (prefix,)
+        self.passthru_prefixes += (re.compile(prefix),)
```

Comment:
This seems like it could fail in scenarios where the pass through prefix contains re metacharacters. Having an re pattern passed in as a parameter avoids backwards compatibility issues and makes the user's intent very clear, as str and re patterns are different types.

Reply:
Yes, you are right.
I've changed the PR with your idea.